src/vcpkg/binarycaching.cpp

```diff
@@ -988,38 +993,36 @@ namespace
         void acquire_zips(View<const InstallPlanAction*> actions,
                           Span<Optional<ZipResource>> out_zip_paths) const override
         {
-            for (size_t idx = 0; idx < actions.size(); ++idx)
-            {
-                auto&& action = *actions[idx];
-                const auto& abi = action.package_abi().value_or_exit(VCPKG_LINE_INFO);
-                auto tmp = make_temp_archive_path(m_buildtrees, action.spec);
-                auto res = m_tool->download_file(make_object_path(m_prefix, abi), tmp);
-                if (res)
-                {
-                    out_zip_paths[idx].emplace(std::move(tmp), RemoveWhen::always);
-                }
-                else
-                {
-                    out_sink.println_warning(res.error());
-                }
-            }
+            parallel_transform(
+                actions, out_zip_paths.begin(), [&](const InstallPlanAction* plan) -> Optional<ZipResource> {
+                    const auto& abi = plan->package_abi().value_or_exit(VCPKG_LINE_INFO);
+                    auto tmp = make_temp_archive_path(m_buildtrees, plan->spec);
+                    auto res = m_tool->download_file(make_object_path(m_prefix, abi), tmp);
+                    if (res)
+                    {
+                        return ZipResource{ZipResource(std::move(tmp), RemoveWhen::always)};
+                    }
+                    else
+                    {
+                        out_sink.println_warning(res.error());
```

Comment:
Missed prints not being synchronised currently in previous assessment.
* https://github.com/microsoft/vcpkg-tool/pull/1290
* https://github.com/microsoft/vcpkg-tool/pull/1323

 This particular print could be extracted relatively easily and is the only print inside the parallel execution.